statsd/pipe.go

```diff
@@ -0,0 +1,9 @@
+// +build !windows
+
+package statsd
+
+import "errors"
+
+func newWindowsPipeWriter(pipepath string) (statsdWriter, error) {
+	return nil, errors.New("Windows Named Pipes are not supported on other operating systems than Windows")
```

Comment:
Consider rewording this for clarity.
```suggestion
	return nil, errors.New("Windows Named Pipes are only supported on Windows")
```